test/acceptance/tests.py

```diff
@@ -592,6 +623,210 @@ def test_file_upload(self):
+        self.staff_area_page = StaffAreaPage(self.browser, self.problem_loc)
+
+    def login_user(self, learner, email=LEARNER_EMAIL, password=LEARNER_PASSWORD):
+        """
+        Logs in an already existing user.
+
+        Args:
+            learner (str): the username of the user.
+            email (str): email (if not specified, LEARNER_EMAIL is used).
+            password (str): password (if not specified, LEARNER_PASSWORD is used).
+        """
+        auto_auth_page = AutoAuthPage(
+            self.browser, email=email, password=password, username=learner,
+            course_id=self.TEST_COURSE_ID, staff=True
+        )
+        auto_auth_page.visit()
+
+    def do_submission(self, email, password):
+        """
+        Creates a user and submission.
+
+        Args:
+            email (str): email for the new user
+            password (str): password for the new user
+
+        Returns:
+            str: the username of the newly created user
+        """
+        auto_auth_page = AutoAuthPage(
+            self.browser, email=email, password=password, course_id=self.TEST_COURSE_ID, staff=True
+        )
+        auto_auth_page.visit()
+        username = auto_auth_page.get_username()
+        self.submission_page.visit().submit_response(self.SUBMISSION)
+
+        return username
+
+    def do_submission_training_self_assessment(self, email, password):
+        """
+        Creates a user and then does submission, training, and self assessment.
+
+        Args:
+            email (str): email for the new user
+            password (str): password for the new user
+
+        Returns:
+            str: the username of the newly created user
+        """
+        username = self.do_submission(email, password)
+        self.do_training()
+        self.submit_self_assessment(self.SELF_ASSESSMENT)
+
+        return username
+
+    def verify_staff_area_fields(self, username, peer_assessments, submitted_assessments, self_assessment):
+        """
+        Verifies the expected entries in the staff area for peer assessments,
+        submitted assessments, and self assessment.
+
+        Args:
+            username (str): the username of the learner to check
+            peer_assessments: the expected fields in the peer assessment section
+            submitted_assessments: the expected fields in the submitted assessments section
+            self_assessment: the expected fields in the self assessment section
+        """
+        self.staff_area_page.visit()
+        self.staff_area_page.show_learner(username)
+        self.staff_area_page.expand_learner_report_sections()
+        self.assertEqual(peer_assessments, self.staff_area_page.status_text('peer__assessments'))
+        self.assertEqual(submitted_assessments, self.staff_area_page.status_text('submitted__assessments'))
+        self.assertEqual(self_assessment, self.staff_area_page.status_text('self__assessment'))
+
+    def test_staff_override_at_end(self):
```

Comment:
Never mind. @cahrens and I spoke about this and this isn't what we had in mind. This is the most complex scenario so should give us confidence in the simpler ones too.